Patch-Source: https://github.com/rpm-software-management/distribution-gpg-keys/commit/ffca1609e48643e0b88e089a501319810b622db2
---
From ffca1609e48643e0b88e089a501319810b622db2 Mon Sep 17 00:00:00 2001
From: Achill Gilgenast <achill@achill.org>
Date: Tue, 2 Sep 2025 09:32:54 +0200
Subject: [PATCH] alpine-linux: Add loongarch64 key

Loongarch64 is a new architecture in Alpine, for which the keys of the
builder were added in 2024-08-14.

Copied from
https://git.alpinelinux.org/aports/tree/main/alpine-keys/alpine-devel@lists.alpinelinux.org-66ba20fe.rsa.pub
(see SOURCES.md)
---
 ...ne-devel@lists.alpinelinux.org-66ba20fe.rsa.pub | 14 ++++++++++++++
 1 file changed, 14 insertions(+)
 create mode 100644 keys/alpine-linux/alpine-devel@lists.alpinelinux.org-66ba20fe.rsa.pub

diff --git a/keys/alpine-linux/alpine-devel@lists.alpinelinux.org-66ba20fe.rsa.pub b/keys/alpine-linux/alpine-devel@lists.alpinelinux.org-66ba20fe.rsa.pub
new file mode 100644
index 0000000000..e8bb47600f
--- /dev/null
+++ b/keys/alpine-linux/alpine-devel@lists.alpinelinux.org-66ba20fe.rsa.pub
@@ -0,0 +1,14 @@
+-----BEGIN PUBLIC KEY-----
+MIICIjANBgkqhkiG9w0BAQEFAAOCAg8AMIICCgKCAgEAtfB12w4ZgqsXWZDfUAV/
+6Y4aHUKIu3q4SXrNZ7CXF9nXoAVYrS7NAxJdAodsY3vPCN0g5O8DFXR+390LdOuQ
++HsGKCc1k5tX5ZXld37EZNTNSbR0k+NKhd9h6X3u6wqPOx7SIKxwAQR8qeeFq4pP
+rt9GAGlxtuYgzIIcKJPwE0dZlcBCg+GnptCUZXp/38BP1eYC+xTXSL6Muq1etYfg
+odXdb7Yl+2h1IHuOwo5rjgY5kpY7GcAs8AjGk3lDD/av60OTYccknH0NCVSmPoXK
+vrxDBOn0LQRNBLcAfnTKgHrzy0Q5h4TNkkyTgxkoQw5ObDk9nnabTxql732yy9BY
+s+hM9+dSFO1HKeVXreYSA2n1ndF18YAvAumzgyqzB7I4pMHXq1kC/8bONMJxwSkS
+Ym6CoXKyavp7RqGMyeVpRC7tV+blkrrUml0BwNkxE+XnwDRB3xDV6hqgWe0XrifD
+YTfvd9ScZQP83ip0r4IKlq4GMv/R5shcCRJSkSZ6QSGshH40JYSoiwJf5FHbj9ND
+7do0UAqebWo4yNx63j/wb2ULorW3AClv0BCFSdPsIrCStiGdpgJDBR2P2NZOCob3
+G9uMj+wJD6JJg2nWqNJxkANXX37Qf8plgzssrhrgOvB0fjjS7GYhfkfmZTJ0wPOw
+A8+KzFseBh4UFGgue78KwgkCAwEAAQ==
+-----END PUBLIC KEY-----
